src/layers/repository/keys.py

```diff
@@ -3,26 +3,31 @@
 
```

Comment:
i was thinking to change this to the following, to make the repo easier to follow, without some magic string and making the pk/sk and associated methods more explicit:

```python
from enum import StrEnum
from typing import Generator

KEY_SEPARATOR = "#"


class TableKeys(StrEnum):
    DEVICE = "D"
    DEVICE_KEY = "DK"
    DEVICE_RELATIONSHIP = "DR"
    PRODUCT_TEAM = "PT"
    ODS_ORGANISATION = "O"

    def key(self, *args) -> str:
        return KEY_SEPARATOR.join((self, *args))

    def filter(
        self, iterable: list[dict[str, str]], key: str
    ) -> Generator[dict[str, str], None, None]:
        return (v for v in iterable if v[key].startswith(f"{self}{KEY_SEPARATOR}"))


def strip_key_prefix(key: str):
    _, tail = key.split(KEY_SEPARATOR, 1)
    return tail
```